### ISSUE TITLE
Still getting 'address already in use' using --inspect on typescript projects

### ISSUE THREAD

**Issue by mjyoung** [vercel/next.js#11821]
# Bug report
When using `NODE_OPTIONS='--inspect' next dev -p 1337` I'm still getting the error:
`Starting inspector on 127.0.0.1:9229 failed: address already in use`

## Describe the bug

From what I understand, this bug should have been fixed as of https://github.com/zeit/next.js/pull/11041. 

However, I still get the error:
```bash
npm run dev

> frontend@0.1.0 dev 
> NODE_OPTIONS='--inspect' next dev -p 1337

Debugger listening on ws://127.0.0.1:9229/90c3d543-273f-492c-a8f6-bb3afd978e64
For help, see: https://nodejs.org/en/docs/inspector
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:1337 ...

Starting inspector on 127.0.0.1:9229 failed: address already in use
```

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Edit `package.json`'s `dev` script:
```
 "scripts": {
    "dev": "NODE_OPTIONS='--inspect' next dev -p 1337",
    ...
  },
  "dependencies": {
    "next": "9.3.4",
    ...
  },
```
2. `npm run dev` or `yarn dev`
3. See error
```
npm run dev

> frontend@0.1.0 dev 
> NODE_OPTIONS='--inspect' next dev -p 1337

Debugger listening on ws://127.0.0.1:9229/90c3d543-273f-492c-a8f6-bb3afd978e64
For help, see: https://nodejs.org/en/docs/inspector
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:1337 ...

Starting inspector on 127.0.0.1:9229 failed: address already in use
```
## Expected behavior

Do not see `address already in use` error

## System information

- OS: macOS
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: `9.3.4` (also tried on `9.3.5-canary.7`)
- Version of Node.js: `12.16.2`

**Comment by ingoclaro** [comment]
could you give a link to a repo that doesn't work? I just tried this and it worked for me. Perhaps the address is really in use, can you check with netstat?

**Comment by PeoB** [comment]
Had same issue, tried with 9.3.4 and 9.3.5-canary.7.

Ended up simply doing `node --inspect node_modules/next/dist/bin/next dev` which seems to do the trick for me

**Comment by mjyoung** [comment]
@ingoclaro I think this may have to do with typescript? I cloned down the `with-typescript-app` example

I did `yarn create next-app --example with-typescript with-typescript-app` and then changed the `dev` script to use `NODE_OPTIONS='--inspect'` and got the 9229 already in use error

**Comment by mjyoung** [comment]
Modified @PeoB's comment into:
```
"dev": "node --inspect ./node_modules/.bin/next dev -p 1337"
``` 

Can confirm this worked for me. No more 9229 already in use business

**Comment by GabrielDelepine** [comment]
I am also having the warning in `9.3.5-canary.6`

![Screenshot 2020-04-10 at 18 39 43](https://user-images.githubusercontent.com/1979087/79007384-5e25d700-7b5b-11ea-8827-035a234fb9c1.png)

Here is my VSCODE debug config
```json
{
    "type": "node",
    "request": "launch",
    "name": "Next: npm dev",
    "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/next",
    "env": {
        "NODE_OPTIONS": "--inspect=9669"
    },
    "port": 9669,
    "console": "integratedTerminal"
},
```

**Comment by Janpot** [comment]
I opened https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/pull/405 to get around this.
You can also use
```
NODE_OPTIONS='--inspect=0'
```
to assign a random port. You'll have to manually configure devtools with this port afterwards though.

**Comment by wei** [comment]
Thanks @Janpot , making the same file change as your [TypeStrong/fork-ts-checker-webpack-plugin#405](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/pull/405/files) fixed it for me!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
